lib/cppcheck.cpp

```diff
@@ -1221,7 +1230,7 @@ unsigned int CppCheck::checkFile(const FileWithDetails& file, const std::string
                     mLogger->setAnalyzerInfo(nullptr);
                 return mLogger->exitcode();
             } catch (const InternalError &e) {
-                ErrorMessage errmsg = ErrorMessage::fromInternalError(e, &tokenizer.list, file.spath());
+                ErrorMessage errmsg = ErrorMessage::fromInternalError(e, nullptr, file.spath());
```

Comment:
Since `TokenList` might throw `InternalError` it seem strange that we pass that object into the exception handling. Also it was no longer available.